lib/i18n/backend/active_record.rb

```diff
@@ -4,9 +4,24 @@
 module I18n
   module Backend
     class ActiveRecord
-      autoload :Missing,     'i18n/backend/active_record/missing'
-      autoload :StoreProcs,  'i18n/backend/active_record/store_procs'
-      autoload :Translation, 'i18n/backend/active_record/translation'
+      autoload :Missing,       'i18n/backend/active_record/missing'
+      autoload :StoreProcs,    'i18n/backend/active_record/store_procs'
+      autoload :Translation,   'i18n/backend/active_record/translation'
+      autoload :Configuration, 'i18n/backend/active_record/configuration'
+
+      class << self
+        def configure(&block)
+          yield(config) if block_given?
+        end
+
+        def config
+          @config ||= Configuration.new
+        end
+
+        def config=(x)
```

Comment:
Yeah I guess my comment from above in regards to your `@_config` suggestion got hidden. I'll paste it here:

Yeah, same thing. Investigating a little further, it's class_variable_set that's throwing it, and otherwise it's allowed to be @config (one @ instead of two). I was aiming to avoid exposing a writer for the config on ActiveRecord, but exposing one and using it over class_variable_set in the tests accomplishes what you're asking for, if you don't mind the setter being there.

I'll make that change and push so you can see it.